metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name":    "simp-fips",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
If you make the parameter change, this will need to bump to `1.0.0` and the `simp.spec` in `simp-core` will need to be updated.